packages/amplify-cli/src/plugin-manager.ts

```diff
@@ -133,10 +133,8 @@ export function getAllPluginNames(pluginPlatform: PluginPlatform): Set<string> {
 }
 
 export async function scan(pluginPlatform?: PluginPlatform): Promise<PluginPlatform> {
-  print.info('Scanning for plugins...');
   try {
     const result = await scanPluginPlatform(pluginPlatform);
-    print.info('Plugin scan successful');
     return result;
   } catch (e) {
     print.error('Plugin scan failed.');
```

Comment:
since we're taking the time to improve the messages, if this fails what is the customer impact and/or recovery story here?

Let's say as a customer, I see this `Plugin scan failed.` in my console. What do I do? What are the consequences? Is there a troubleshooting (if so, can we link the URL here?)

Reply:
+1: Kind of fails our "Expose friendly error messages to the customer that assist with debugging" tenet

Reply:
Discussed offline with other reviewers. The error rate is practically zero on this.